lib/ain-cpp-imports/src/lib.rs

```diff
@@ -61,13 +61,21 @@ mod ffi {
     pub fn pastChangiIntermediateHeight3() -> bool {
         unimplemented!("{}", UNIMPL_MSG)
     }
+    pub fn getClientVersion() -> [u64; 3] {
+        unimplemented("{}", UNIMPL_MSG)
+    }
 }
 
 pub fn get_chain_id() -> Result<u64, Box<dyn Error>> {
     let chain_id = ffi::getChainId();
     Ok(chain_id)
 }
 
+pub fn get_client_version() -> Result<[u64; 3], Box<dyn Error>> {
+    let version = ffi::getClientVersion();
```

Comment:
Can return `ffi::getClientVersion()` without wrapping in a Result since it does not throw

Reply:
Can this said of the same for the other functions in the `ffi` bridge such as `get_chain_id()`?

Reply:
@andyrobert3 Good spot. This should be a refactor after RC.